docs/framework/guides/tutorials/implementing-an-inline-widget.md

```diff
@@ -462,11 +462,11 @@ export default class PlaceholderEditing extends Plugin {
 	}
 
 	_defineSchema() {
-		// ...
+		// Previously registered schema.
 	}
 
 	_defineConverters() {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.